metering/metering-uninstall.adoc

```diff
@@ -1,13 +1,22 @@
 :context: metering-uninstall
 [id="metering-uninstall"]
-= Uninstalling Metering
+= Uninstalling metering
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can remove metering from your {product-title} cluster.
+Uninstall metering by removing the Metering Operator and associated metering components from your {product-title} cluster and uninstalling the Operator from your metering namespace.
```

Comment:
I think we may have this reversed:
- We remove the Metering Operator from the cluster by following the xref link, which deletes the Metering Operator.
- We remove the associated metering components by deleting the metering namespace.